packages/clients/dish/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # CHANGELOG
 
+## 1.0.1
+
+- Fix: Remove configurable attribute windowLength which does not exist anymore.
+
```

Comment:
Changelog entry is not necessary, as the respective customer does not recognize that parameter.

Reply:
thanks for the note 🏓